### DISCUSSION TITLE
Feature Request: Pass URL Parameters from Short URL to Long URL

### DISCUSSION THREAD

**Discussion by WanderingApps** [YOURLS/YOURLS#3717] [Ideas]
Hello,

I have a use case for any custom URL parameter to be passed from the short URL to my long url where it would be most helpful if any URL arguments that were applied to the short URL would be applied to the long URL.

  mysite.com/short?user=mike
  mysite.com/prod/assets/long_url_destination.php?user=mike

I didn't see any options for this in the forum and can understand that it does make the short URL longer so maybe not a common request.

**Comment by dgw** [comment]
There are no less than 6 plugins submitted to https://github.com/YOURLS/awesome that implement some version of this feature. <kbd>Ctrl</kbd>+<kbd>F</kbd> for `query`
